build/media-suite_compile.sh

```diff
@@ -655,6 +666,11 @@ if [[ $ffmpeg != no ]] && enabled libzimg &&
     do_checkIfExist
 fi
 
+if [[ $exitearly = EE3 ]]; then
+    do_simple_print -p '\n\t'"${orange}Exit due to env var MABS_EXIT_EARLY set to EE3"
+    return
+fi
+
 set_title "compiling audio tools"
 do_simple_print -p '\n\t'"${orange}Starting $bits compilation of audio tools${reset}"
```

Comment:
I wonder if it would be better to just split the `build()` function into multiple sub-functions for each section and keep the `build()` function as a wrapper and consolidate the early exit checks into that wrapper instead